lib/ansible/modules/storage/ibm/ibm_sa_host_ports.py

```diff
@@ -109,9 +109,14 @@ def main():
     port_exists = False
     ports = [port.get('port_name') for port in ports]
 
+    fc_ports = (module.params.get('fcaddress')
```

Comment:
```suggestion
    fc_ports = module.params.get('fcaddress', [])
```

Reply:
@Akasurde , During our testing we found that 'None' is assigned to fcaddress' if 'fcaddress' is not given in the playbook. Thus, in our case 'fc_ports' will get 'None' rather than []